catalog/datasets/muziekschatten-onderwerpen.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.muziekschatten.nl/onderwerpen",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Muziekschatten - onderwerpen"
+    }
+  ],
+  "alternateName": [
+    {
+      "@language": "nl",
+      "@value": "SOM"
```

Comment:
Is the source Muziekschatten: onderwerpen indeed abbreviated as SOM? Or would that only be the alternateName of the creator, i.e. Stichting Omroep Muziekbibliotheek. Please note that you can leave out the alternateName if you want to. E.g. RKDArtists is just RKDArtists, without any abbreviation.

Reply:
The precise relation between Muziekschatten and SOM is not completely clear to me. It looks like the SOM data is a subcollection of Muziekschatten, see also http://www.muziekbibliotheekvandeomroep.nl/. I left the SOM value in here in order to make a reference this and to the dataset in the previous version op the prototype. In that version the source is called Stichting Omroep Muziekbibliotheek and it points to the muziekschatten endpoint. As far as I know SOM doesn't exists any more and it's collection is part of the Muziekschatten service.

Reply:
Interesting! Creator https://www.muziekschatten.nl is indeed called Stichting Omroep Muziek (SOM). The [source itself](http://data.muziekschatten.nl/som/) seems to be called Stichting Omroep Muziek Bladmuziekcatalogus. Are we selecting only subjects (‘onderwerpen’) from that source? Shouldn’t we also select artists and songs?

@sdevalk You previously worked with them. Do you have a suggestion how to name both this source its creator?

Reply:
> @sdevalk You previously worked with them. Do you have a suggestion how to name both this source its creator?

I'll ask SOM!

Reply:
I had contact with SOM:

1. The `name` of the organization/publisher should be `Stichting Omroep Muziek`, the `alternateName` should be `SOM` and the `@id` should be `https://www.omroepmuziek.nl/`

2. The `name` of the dataset should be/contain `Muziekschatten`. Or in case of the 'onderwerpen': `Muziekschatten: onderwerpen`.

Reply:
Changed the definitions as suggested above.